models/psf.py

```diff
@@ -150,6 +150,11 @@ def info( self ):
     def info( self, value ):
         self._info = value
 
+    @property
+    def image_shape(self):
+        """The shape of the image this PSF is for."""
+        return self.header['NAXIS2'], self.header['NAXIS1']
```

Comment:
This isn't right.  For PSFs from PSFex, the header is the header of the .psf FITS file written by PSFex (or so is documented in the "header" property), and the NAXIS1/NAXIS2 there are the size of the FITS data in the PSF, which is very different from the image that the PSF is measured for.

`self.image.header['NAXIS2'], self.image.header['NAXIS1']` should do it.

Reply:
I wanted to allow the PSF object to independently know the size of the image it is related to, without having to load it from the database (and indeed, load it from file to read the header). 
I'm not sure if this is necessary anymore as I get a fairly good PSF when just requesting 0,0 as the coordinates.